README.md

```diff
@@ -34,29 +34,25 @@
 ```jsx
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { createStore, combineReducers, applyMiddleware } from 'redux';
+import { createStore } from 'redux';
 import { createInputsReducer, connectWithInputs, ReduxInputsWrapper } from 'redux-inputs';
 import { Provider } from 'react-redux';
-import thunk = from 'redux-thunk';
```

Comment:
redux-thunk is just a requirement of using the library. Having to go and change configuration to use some of the features is oddly inconsistent.